usagov/vote-gov-drupal
# VOTE-695: Update Drupal core, Drush, and contrib modules.

## Jira ticket

[VOTE-695](https://cm-jira.usa.gov/browse/VOTE-695)

## Description

Upgrade Drupal core and contrib modules to current versions.

## Deployment and testing

### Pre-deploy

1. Insert any required steps to take before merging and running post-deploy.

### Post-deploy

1. Locally, run`lando retune -y`

### QA/Test

1. Browse site locally to confirm that everything works as expected.
2. Login as an administrator, view Available Updates page (/admin/reports/updates) and confirm that modules and core are updated to latest versions.
3. Review Status Report page (/admin/reports/status) and confirm no significant errors or warnings are listed.

1. On cloud environments, confirm that media file uploads work and are accessible via S3 path when clicking on the link for the file from the media files admin list page.

## Checklist for the Developer




## Checklist for the Peer Reviewers
